ci-operator/jobs/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main-periodics.yaml

```diff
@@ -18,6 +18,14 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-parallel-conformance
+  reporter_config:
+    slack:
+      channel: '#image-based-ci'
+      job_states_to_report:
+      - failure
+      - success
```

Comment:
Why do we want to be reported for the successful runs?

Reply:
I plan to remove this in the future, a success is basically non-existant at the moment